### PR TITLE
Change Log Level for Trim messages in Stream View

### PR DESCRIPTION
**Pull request by annym**
## Overview

Modify log levels for trimmed messages that are not really critical until they lead to a TAE which is already logged as an error. It is common for trim message "warnings" to happen once before a valid checkpoint is loaded.

### CODE REVIEW

File: runtime/src/main/java/org/corfudb/runtime/view/stream/AddressMapStreamView.java [hunk 1]
```diff
@@ -161,11 +161,12 @@ protected boolean discoverAddressSpace(final UUID streamId,
...
                                     "loaded checkpoint with start address %s, while accessing the " +
-                                    "stream at version %s", this, streamAddressSpace.getTrimMark(),
+                                    "stream at version %s. Looking for a new checkpoint.", this,
+                            streamAddressSpace.getTrimMark(),
                             getCurrentContext().checkpointSuccessStartAddr, maxGlobal);
-                    log.warn(message);
+                    log.info(message);
                     if (options.ignoreTrimmed) {
-                        log.warn("getStreamAddressMap[{}]: Ignoring trimmed exception for address[{}].",
+                        log.debug("getStreamAddressMap[{}]: Ignoring trimmed exception for address[{}].",
```
Comment (epalta) [inline runtime/src/main/java/org/corfudb/runtime/view/stream/AddressMapStreamView.java]: Did you figure out why we were seeing the huge number at the end of the message ?
2019-06-27T03:26:49.390Z WARN CorfuDB-DCN-Publisher-0 AddressMapStreamView - getStreamAddressMap[{ae2a@448367}] stream has been trimmed at address 407912 and this space is not covered by the loaded checkpoint with start address -1, while accessing the stream at version 9223372036854775807

Comment (annym) [reply runtime/src/main/java/org/corfudb/runtime/view/stream/AddressMapStreamView.java]: Yes, that is when we do remaining() which goes up to Long.MAX

File: runtime/src/main/java/org/corfudb/runtime/view/stream/AddressMapStreamView.java [hunk 2]
```diff
@@ -250,7 +251,7 @@ private boolean processCheckpointBatchByEntry(List<Long> batch,
         } catch (TrimmedException ste) {
             // The valid checkpoint has been trimmed.
             if (options.ignoreTrimmed) {
-                log.warn("processCheckpoint[{}]: Ignoring trimmed exception for address[{}]," +
+                log.debug("processCheckpoint[{}]: Ignoring trimmed exception for address[{}]," +
```
Comment (medhavidhawan) [inline runtime/src/main/java/org/corfudb/runtime/view/stream/AddressMapStreamView.java]: In the case of ignoreTrimmed is this misleading ?

Comment (annym) [reply runtime/src/main/java/org/corfudb/runtime/view/stream/AddressMapStreamView.java]: That's why we are changing the log level. As it only ignores it if it was explicitly set to be ignored. Shouldn't then be something to worry about or notify. If ignored trimmed flag is set,  this stream is never reset to load from a checkpoint which means that this message is likely to appear a lot of times.

**Pull request merged**